src/CommunityToolkit.Aspire.OllamaSharp/AspireOllamaSharpExtensions.cs

```diff
@@ -23,11 +22,11 @@ public static class AspireOllamaSharpExtensions
     /// <param name="connectionName">A name used to retrieve the connection string from the ConnectionStrings configuration section.</param>
     /// <param name="configureSettings">An optional delegate that can be used for customizing options. It's invoked after the settings are read from the configuration.</param>
     /// <exception cref="UriFormatException">Thrown when no Ollama endpoint is provided.</exception>
-    public static void AddOllamaApiClient(this IHostApplicationBuilder builder, string connectionName, Action<OllamaSharpSettings>? configureSettings = null)
+    public static AspireOllamaApiClientBuilder AddOllamaApiClient(this IHostApplicationBuilder builder, string connectionName, Action<OllamaSharpSettings>? configureSettings = null)
```

Comment:
This is a breaking change. Are you OK with that?

Reply:
Yes, it is a breaking change, but in terms of breaking changes it's a pretty unobtrusive one. Having it as a `void` method previously means that you weren't capturing anything to now having a returned value, but since C# doesn't require return assignment it shouldn't result in a compilation error.

It would be a problem if anyone is using reflection to find the method and invoking it based off an expected signature, and _maybe_ a problem in F# as the return needs to be handled (or piped to `ignore`), but both are pretty obscure edge cases.